plugins/imklog/imklog.c

```diff
@@ -301,6 +306,8 @@ CODESTARTbeginCnfLoad
 	pModConf->bKeepKernelStamp = 0;
 	pModConf->iFacilIntMsg = klogFacilIntMsg();
 	loadModConf->configSetViaV2Method = 0;
+	pModConf->ratelimitBurst = 10000; /* arbitrary high limit */
```

Comment:
the default should be inline with what we currently have (no ratelimit). Else we break existing configuration.

Reply:
The default is to have the ratelimitInterval on 0 which will result in the current behaviour (no ratelimit.)

I can't set ratelimitBurst to 0, or an assert at runtime/ratelimit.c:160 will fail.  The alternative is to wrap the two lines after CODESTARTactivateCnf at plugins/imklog/imklog.c in an if to prevent the ratelimiter from being initialised.